python/lsst/daf/butler/cli/utils.py

```diff
@@ -789,7 +790,7 @@ class MWOptionDecorator:
     """
 
     def __init__(self, *param_decls: Any, **kwargs: Any) -> None:
-        self.partialOpt = partial(click.option, *param_decls, cls=partial(MWOption), **kwargs)
+        self.partialOpt = partial(click.option, *param_decls, cls=partial(MWOption), **kwargs)  # type: ignore
```

Comment:
mypy doesn't think that `partial(MWOption)` is the same as an option and so now complains.